src/Open3D/Core/Tensor.h

```diff
@@ -381,39 +407,103 @@ class Tensor {
 
     /// Adds a tensor and returns the resulting tensor.
     Tensor Add(const Tensor& value) const;
+    template <typename T>
+    Tensor Add(T scalar_value) const {
+        return Add(Tensor::Full(shape_, scalar_value, dtype_, GetDevice()));
```

Comment:
instead of shape_ would it be better to specify {1} and just use broadcasting? so that if this is a giant tensor we do not have to allocate and destroy another giant tensor of this size?
(same for all scalar ops below)

Reply:
good catch, changed to broadcasting with tensor shape `{}`